documentation/assemblies/deploying/assembly-deploy-kafka-mirror-maker.adoc

```diff
@@ -6,9 +6,20 @@
 [id='kafka-mirror-maker-{context}']
 = Deploying Kafka MirrorMaker
 
-The Cluster Operator deploys one or more Kafka MirrorMaker replicas to replicate data between Kafka clusters.
-This process is called mirroring to avoid confusion with the Kafka partitions replication concept.
-MirrorMaker consumes messages from the source cluster and republishes those messages to the target cluster.
+[role="_abstract"]
+Kafka MirrorMaker replicates data between two or more Kafka clusters, within or across data centers.
+This process is called mirroring to avoid confusion with the concept of Kafka partition replication.
+MirrorMaker consumes messages from a source cluster and republishes those messages to a target cluster.
+
+Data replication across clusters supports scenarios that require the following:
+
+* Recovery of data in the event of a system failure
+* Aggregation of data for analysis
```

Comment:
What exactly does this mean? How do you aggregate the data in Mirror Maker?

Reply:
I would assume Paul is referring to a scenario where you have a "central" office and subsidiaries. Data from subsidiaries are mirrored to the central one having a topic naming during the mirroring to take the data separated but then running applications that can get data from the central one only and aggregate them.
It's a valid use case, maybe we should be clearer in this one line description.

Reply:
I guess we're replicating data from multiple source clusters.
I changed to `Consolidation of data from multiple source clusters for centralized analysis`